Heretic drip and toys

airlines7 opened the pull request:
### TL;DR
Adds toy heretic sickles to donksoft vending
Adds a "fashionable robe" to the costume autodrobe as contraband, the costume version of the ominous armor.

# Description

### The drippy robe
After all these years the heretic drip now for the common man. I've made sure that it does not decrease damage as it is just a "fashionable" hood and robe. Has some silly flavor text about a massacre at a Nanotrasen fashion show.
Video proof  https://user-images.githubusercontent.com/98609667/236930600-580c9b00-b6a2-4f16-9b83-41228e526487.mp4

![image](https://user-images.githubusercontent.com/98609667/236932658-7718c17a-a962-4a9e-a0a7-33b3e9a79bc8.png)

![image](https://user-images.githubusercontent.com/98609667/236931230-d275fbef-f838-4179-b266-5c85a44c64fd.png)

![image](https://user-images.githubusercontent.com/98609667/236931111-6ea9a5b2-efc4-4f9d-90f1-a79047c9fac1.png)

### The toy blades
I've made sure the toys do no damage and do not paralyze with the hook, I don't know why the sickles show up as random icons. I'll (probably not) figure it out and fix it at some point.

Video Proof  https://user-images.githubusercontent.com/98609667/236933452-6903479e-464e-4732-810a-f2aad86edb14.mp4

![image](https://user-images.githubusercontent.com/98609667/236931013-6eb8c289-912a-4147-965a-389a2c3039d3.png)

![image](https://user-images.githubusercontent.com/98609667/236931789-bc3c20d6-4ae6-48b3-8a7a-d2552eefda52.png)

I see the "metalicr" its already fixed, too lazy to upload another photo demo.
The sickles also don't do the honk I coded in, but that just makes it more tolerable as a toy. Feel free to remove or fix.




# Changelog

:cl:  Airlines7
rscadd: Heretic costumes for the comedic clown and cynical mime.
rscadd: Toy Heretic blades, brought to you by Donksoft.

/:cl:

airlines7 committed:
vending-additions

airlines7 committed:
heretic-toy-blades

airlines7 committed:
fake-robes

currently has the glowey animation, this can be easily changed if deemed too hard to discern.

airlines7 committed:
"metalicr" fix

airlines7 committed:
Grammar >:(

airlines7 committed:
Makes heretic robe contraband

Makes more sense with the flavor text, also makes it a teeny tiny bit harder to get to so people don't abuse it to annoy sec.

airlines7 committed:
forgor ro remove comma

airlines7 committed:
more grammar

added cult sickles to list on the top of the file

airlines7 committed:
toy blades and book work with fake robe

MajManatee reviewed (changes requested):
Fix gramatical issues, fix the icon being a cup.

airlines7 committed:
Last of the grammer

airlines7 committed:
Merge branch 'yogstation13:master' into master

airlines7 committed:
Workaround for vending icons

This fixes the random icons, I also added the chefdrobe so those get fixed too.
Adds the donksoft vending and chefdrobe to CC in order to fix the icons in a location by the testing area.
Thank you to Blundir and offbeatwitch for the horrible idea

airlines7 committed:
>:(

airlines7 committed:
remove hook

airlines7 committed:
resync

airlines7 committed:
Merge branch 'yogstation13:master' into master

airlines7 committed:
remove stray line

airlines7 committed:
whoops wrong file

airlines7 committed:
Add files via upload

airlines7 committed:
Merge branch 'yogstation13:master' into master

airlines7 committed:
remove hook for realsies

airlines7 committed:
Readd hook

No longer brings people to the user.

airlines7 committed:
Reconvert CC to LF
